src/mcts/search.cc

```diff
@@ -1027,9 +1126,13 @@ void SearchWorker::InitializeIteration(
     std::unique_ptr<NetworkComputation> computation) {
   computation_ = std::make_unique<CachingComputation>(std::move(computation),
                                                       search_->cache_);
+  computation_->Reserve(params_.GetMiniBatchSize());
   minibatch_.clear();
+  minibatch_.reserve(2 * params_.GetMiniBatchSize());
 }
 
+#define USE_WORKERS 1
+
 // 2. Gather minibatch.
 // ~~~~~~~~~~~~~~~~~~~~
 void SearchWorker::GatherMinibatch() {
```

Comment:
Do I understand correctly that only one (SearchWorker)thread may be inside `GatherMinibatch()` at the same time?
If yes, how do we ensure that? it seems that the mutex is gone.
If no, how do we protect multiple gatherers from touching the same nodes?

Reply:
Disregard this question, found lock in `PickNodesToExtend()`